Google.Api.Generator.Tests/ProtoTests/Lro/LroClient.cs

```diff
@@ -0,0 +1,135 @@
+        /// <returns>A Task containing the RPC response.</returns>
+        public virtual stt::Task<lro::Operation<LroResponse, LroMetadata>> ResourcedMethodAsync(string name, gaxgrpc::CallSettings callSettings = null) =>
+            ResourcedMethodAsync(new ResourceRequest
+            {
+                Name = name ?? "",
+            }, callSettings);
+
+        /// <summary>
+        /// Test an LRO RPC with a method signature that contains resource-names.
+        /// </summary>
+        /// <param name="name">
+        /// </param>
+        /// <param name="cancellationToken">A <see cref="st::CancellationToken"/> to use for this RPC.</param>
+        /// <returns>A Task containing the RPC response.</returns>
+        public virtual stt::Task<lro::Operation<LroResponse, LroMetadata>> ResourcedMethodAsync(string name, st::CancellationToken cancellationToken) =>
+            ResourcedMethodAsync(name, gaxgrpc::CallSettings.FromCancellationToken(cancellationToken));
+
+        /// <summary>
+        /// Test an LRO RPC with a method signature that contains resource-names.
+        /// </summary>
+        /// <param name="name">
+        /// </param>
+        /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
+        /// <returns>The RPC response.</returns>
+        public virtual lro::Operation<LroResponse, LroMetadata> ResourcedMethod(ResourceName name, gaxgrpc::CallSettings callSettings = null) =>
+            ResourcedMethod(new ResourceRequest
+            {
+                ResourceName = name,
+            }, callSettings);
+
+        /// <summary>
+        /// Test an LRO RPC with a method signature that contains resource-names.
+        /// </summary>
+        /// <param name="name">
+        /// </param>
+        /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
+        /// <returns>A Task containing the RPC response.</returns>
+        public virtual stt::Task<lro::Operation<LroResponse, LroMetadata>> ResourcedMethodAsync(ResourceName name, gaxgrpc::CallSettings callSettings = null) =>
+            ResourcedMethodAsync(new ResourceRequest
+            {
+                ResourceName = name,
+            }, callSettings);
+
+        /// <summary>
+        /// Test an LRO RPC with a method signature that contains resource-names.
+        /// </summary>
+        /// <param name="name">
+        /// </param>
+        /// <param name="cancellationToken">A <see cref="st::CancellationToken"/> to use for this RPC.</param>
+        /// <returns>A Task containing the RPC response.</returns>
+        public virtual stt::Task<lro::Operation<LroResponse, LroMetadata>> ResourcedMethodAsync(ResourceName name, st::CancellationToken cancellationToken) =>
+            ResourcedMethodAsync(name, gaxgrpc::CallSettings.FromCancellationToken(cancellationToken));
+        // TEST_END
+    }
```

Comment:
I seem to remember we used to generate a method to fetch an operation based on its name as well... is that coming in another PR?

Reply:
See previous comment. The methods you mention are already tested in the `BasicLro` tests.